worker/app/hw_detect.py

```
@@ -14,52 +19,184 @@
+    # Respect user override
+    if "LIBVA_DRIVER_NAME" in os.environ:
+        return os.environ["LIBVA_DRIVER_NAME"]
+
+    for driver in ("iHD", "i965", ""):
+        env = os.environ.copy()
+        if driver:
+            env["LIBVA_DRIVER_NAME"] = driver
+
+        # Try vainfo first
+        try:
+            result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)